applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -24,49 +24,81 @@ LOG_MODULE_REGISTER(bis_headset, CONFIG_BLE_LOG_LEVEL);
 BUILD_ASSERT(CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT <= 2,
 	     "A maximum of two broadcast streams are currently supported");
 
+struct audio_codec_info {
+	uint8_t id;
+	uint16_t cid;
+	uint16_t vid;
+	int frequency;
+	int frame_duration_us;
+	int chan_allocation;
+	int octets_per_sdu;
+	int bitrate;
+	int blocks_per_sdu;
+};
+
+struct active_stream {
+	struct bt_audio_stream *stream;
+	struct audio_codec_info *codec;
+};
+
 static struct bt_audio_broadcast_sink *broadcast_sink;
 
-static struct bt_audio_stream streams[CONFIG_LC3_DEC_CHAN_MAX];
-static struct bt_audio_stream *streams_p[ARRAY_SIZE(streams)];
+static struct bt_audio_stream audio_streams[CONFIG_BT_AUDIO_BROADCAST_SNK_STREAM_COUNT];
+static struct bt_audio_stream *audio_streams_p[ARRAY_SIZE(audio_streams)];
```

Comment:
This could need a rewrite like we did for cis_gateway. A pointer to the stream should be in a struct with the codec info, like it is in the active stream, and then active stream could just be a pointer, in stead of a struct.

Reply:
Discussed it, will keep it as is and improve later